catalog-info.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: citz-imb-sso-express-(npm-package)
```

Comment:
In the sso react package we dont have the brackets here. I think we should keep them the same.